README.md

```diff
@@ -78,8 +85,7 @@ If you are new to the group, check out our [New Members Page](NEWMEMBERS.md) and
 * SAFE WG - renamed to CNCF Security SIG
 * [(Proposed) CNCF Policy Working Group](/policy-wg-merging.md) - Merged into SAFE WG
 
-## Members list
-
+## Members List
```

Comment:
see [CONTRIBUTING](https://github.com/cncf/sig-security/blob/master/CONTRIBUTING.md) --style guide is sentence cap for headers.   I know it looks a little odd when headline is just two words -- this could be simplified to "Members" :)

Reply:
Got it! it looks like most of the headings have to be corrected :) . also, the contributing guide needs to be updated as well https://github.com/cncf/sig-security/blob/master/CONTRIBUTING.md#reviewing-pull-requests